Online/Menus/CSS/HandleInputsOnCSS.asm

```diff
@@ -457,6 +475,19 @@ branchl r12, HSD_Free
 restore
 blr
 
+################################################################################
+# Function: Reset disconnect button hold timer
+################################################################################
+FN_RESET_DISCONNECT_HOLD_TIMER:
+backup
+
+
+li r3, 0
+stb r3, CSSDT_Z_BUTTON_HOLD_TIMER(REG_CSSDT_ADDR)
```

Comment:
writing to an address that wasn't passed into the function, should pass REG_CSSDT_ADDR into the function as an arg.

Reply:
I think he made it a function to save a few instructions. If he passed it in every time it wouldn't save any and might be better to just not even use a function.

I'd actually probably recommend getting rid of the function and maybe storing zero to a non-volatile register to do like `stb REG_ZERO, CSSDT_Z_BUTTON_HOLD_TIMER(REG_CSSDT_ADDR)`

Reply:
It is exactly what Fizzi said.
I can try the registry though, although I thought the value could change at some point so thats why I went with using the css data table instead.

Reply:
The registry didn't work, so I'm sticking with the helper function unless you guys want the function removed.